catalog/schematron/catalog-schematron-plugin/src/test/java/ddf/services/schematron/SchematronValidationServiceTest.java

```diff
@@ -145,11 +145,17 @@ private SchematronValidationService getService(boolean suppressWarnings, String
 
         ArrayList<String> schemaFiles = new ArrayList<>();
         for (String schematronFile : schematronFiles) {
-            String path = schematronFile;
             if (useClassLoader) {
-                path = SchematronValidationServiceTest.class.getClassLoader().getResource(schematronFile).getPath();
+                URL schematronResource = SchematronValidationServiceTest.class.getClassLoader().getResource(schematronFile);
+                if (schematronResource == null) {
+                    throw new NullPointerException(
+                            "The Schematron Resource came back null. Did you remove the resources folder?");
```

Comment:
Don't put things like "you" in comments. It should say something like "Was the resources folder removed?"